pkg/edit/builtins_test.go

```diff
@@ -208,6 +209,76 @@ var bufferBuiltinsTests = []struct {
 		tk.CodeBuffer{Content: "ab", Dot: 1},
 		tk.CodeBuffer{Content: "a", Dot: 1},
 	},
+	{
+		"transpose-rune with dot at beginning",
+		tk.CodeBuffer{Content: "abc", Dot: 0},
+		tk.CodeBuffer{Content: "bac", Dot: 2},
+	},
+	{
+		"transpose-rune with dot in middle",
+		tk.CodeBuffer{Content: "abc", Dot: 1},
+		tk.CodeBuffer{Content: "bac", Dot: 2},
+	},
+	{
+		"transpose-rune with dot at end",
+		tk.CodeBuffer{Content: "abc", Dot: 3},
+		tk.CodeBuffer{Content: "acb", Dot: 3},
+	},
+	{
+		"transpose-rune with one character and dot at end",
+		tk.CodeBuffer{Content: "a", Dot: 1},
+		tk.CodeBuffer{Content: "a", Dot: 1},
+	},
+	{
+		"transpose-rune with one character and dot at beginning",
+		tk.CodeBuffer{Content: "a", Dot: 0},
+		tk.CodeBuffer{Content: "a", Dot: 0},
+	},
+	{
+		"transpose-word with dot at beginning",
+		tk.CodeBuffer{Content: "ab  bc cd", Dot: 0},
+		tk.CodeBuffer{Content: "bc  ab cd", Dot: 6},
+	},
+	{
+		"transpose-word with dot in between words",
+		tk.CodeBuffer{Content: "ab  bc cd", Dot: 6},
+		tk.CodeBuffer{Content: "ab  cd bc", Dot: 9},
+	},
+	{
+		"transpose-word with dot at end",
+		tk.CodeBuffer{Content: "ab  bc cd", Dot: 9},
+		tk.CodeBuffer{Content: "ab  cd bc", Dot: 9},
+	},
+	{
+		"transpose-word with dot in the middle of a word",
+		tk.CodeBuffer{Content: "ab  bc cd", Dot: 5},
+		tk.CodeBuffer{Content: "bc  ab cd", Dot: 6},
+	},
+	{
+		"transpose-word with one word",
+		tk.CodeBuffer{Content: " ab  ", Dot: 4},
+		tk.CodeBuffer{Content: " ab  ", Dot: 4},
+	},
+	{
+		"transpose-word with no words",
+		tk.CodeBuffer{Content: " \t\n  ", Dot: 4},
+		tk.CodeBuffer{Content: " \t\n  ", Dot: 4},
+	},
+	{
+		"transpose-word with complex input",
+		tk.CodeBuffer{Content: "cd ~/downloads;", Dot: 4},
+		tk.CodeBuffer{Content: "~/downloads;", Dot: 15},
```

Comment:
Fix this test - you should be able to see the test errors.

Reply:
Whoops, sorry.